app/scrcpy.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=scrcpy
+GenericName=Android Remote Control
+Comment=Display and control your Android device
+# For some users, `adb` is not in default $PATH but the one configured in .bashrc/.zshrc/...
+# Run an interactive shell to get the same path used in terminals.
+Exec=/bin/sh -c '"$SHELL" -i -c scrcpy'
```

Comment:
For system-wide installations, Exec should use the system-wide `adb` installed at, e.g.: `/usr/bin/adb` and leave customization concerns up to the user in e.g.: `~/.local/applications/scrcpy.desktop`
:+1: good to merge